README.md

```diff
@@ -17,6 +17,7 @@ What are constructor functions and the `new` keyword? What is a prototype? Descr
 
 Your Answer:
 ```text
+constructor function is a function that helps in creating new multiple similar objects. New keyword is used when creating those multiple objects. In constructor function there are multiple instances of a constructor function. Prototype is an object where other objects can be created. Constructor function has keys prototype does not.  
 
 ```
 
```

Comment:
Couple notes...
* What do you mean by: `In constructor function there are multiple instances of a constructor function. Prototype is an object where other objects can be created.`
* A prototype can have properties. We tend to use them, however, to define methods that are shared by instances of an object.